src/sentry/replays/usecases/ingest/slow_click.py

```diff
@@ -0,0 +1,70 @@
+import datetime
+from typing import Any, Dict
+
+from sentry import options
+from sentry.issues.grouptype import ReplaySlowClickType
+from sentry.replays.usecases.ingest.events import SentryEvent
+from sentry.replays.usecases.issue import new_issue_occurrence
+
+
+def report_slow_click_issue(project_id: int, replay_id: str, event: SentryEvent) -> bool:
+    # Only report slow clicks if the option is enabled.
+    if not options.get("replay.issues.slow_click"):
+        return False
+
+    payload = event["data"]["payload"]
+
+    # Only timeout reasons on <a> and <button> tags are accepted.
+    if payload["data"]["node"]["tagName"] not in ("a", "button"):
+        return False
+    elif payload["data"]["endReason"] != "timeout":
+        return False
+
+    # Seconds since epoch is UTC.
+    timestamp = datetime.datetime.fromtimestamp(payload["timestamp"])
+    timestamp = timestamp.replace(tzinfo=datetime.timezone.utc)
+
+    _report_slow_click_issue(
+        environment="prod",
+        fingerprint=payload["message"],
```

Comment:
what is `message` here? is that the path of the HTML element?

Reply:
Yes, the CSS selector.

Reply:
I wonder how this will end up with duplicate / similar issues / new issues created when small changes are made to the DOM. something to think about for future iterations.

Reply:
I believe @billyvg had a query for this?

Reply:
518 results over unknown timeperiod (this is from bigquery)

Reply:
it might be worth doing a bit of manual prodding of the particular issues now to see if we can find ones that are similar and come up with heuristics to see if we can combine them.

Reply:
and our "similarity" implementation for stacktrace issues for inspiration: https://github.com/getsentry/sentry/blob/7dbd78a6eedd15c46482ceb54eb0fb52f514232c/src/sentry/scripts/similarity/index.lua#L7

Reply:
> The way I'm filtering we should create roughly ~300 issues in 3 weeks.

If that means we have 300 different broken buttons and anchors I think that's fine. Otherwise worth digging further